community/neo4j-harness/src/test/java/org/neo4j/harness/JavaProceduresTest.java

```diff
@@ -163,10 +295,45 @@ public void shouldWorkWithInjectableFromKernelExtension() throws Throwable
             HTTP.Response response = HTTP.POST( server.httpURI().resolve( "db/data/transaction/commit" ).toString(),
                     quotedJson( "{ 'statements': [ { 'statement': 'CALL hello' } ] }" ) );
 
+            assertEquals( "[]", response.get( "errors" ).toString() );
             JsonNode result = response.get( "results" ).get( 0 );
             assertEquals( "result", result.get( "columns" ).get( 0 ).asText() );
             assertEquals( "world", result.get( "data" ).get( 0 ).get( "row" ).get( 0 ).asText() );
-            assertEquals( "[]", response.get( "errors" ).toString() );
         }
     }
+
+    @Test
+    public void shouldWorkWithInjectableFromKernelExtensionWithMorePower() throws Throwable
+    {
+        // When
+        try ( ServerControls server = TestServerBuilders.newInProcessBuilder()
```

Comment:
Is there a way to not use REST for this test?
Just seems strange that we do not develop REST further but use it in tests.